### ISSUE TITLE
Cannot set Text of a Textbox: Error: Member not found

### ISSUE THREAD

**Issue by calgara12** [FlaUI/FlaUI#440]
**Describe the bug**
I am trying to automate the application Bomgar 

I want to set the Text of two Textboxes (highlighted in yellow in the screenshot) with UIAutomation. But i always get the following Error:
System.ComponentModel.Win32Exception: 'Member not found. (Exception from HRESULT: 0x80020003 (DISP_E_MEMBERNOTFOUND))'

**Code snippets**
First attempt:
```
var tbs = bomgarAuthenticationDialog.FindAllDescendants(d => d.ByControlType(ControlType.Edit));
var tb1=tbs[0].AsTextBox().Enter("Test");
```
Second attempt:
```
var tbs = bomgarAuthenticationDialog.FindAllDescendants(d => d.ByControlType(ControlType.Edit));
var tb1=tbs[0].AsTextBox().Text = "Test";
```
Third attempt:
```
var tbs = bomgarAuthenticationDialog.FindAllDescendants(d => d.ByControlType(ControlType.Edit));
var vp=tbs[0].Patterns.Value.Pattern;
vp.SetValue("Test");
```

All attempts result in the same error message that i mentioned above.

**Screenshots**
![image](https://user-images.githubusercontent.com/28657023/128350393-ca729bc3-7f15-4cb7-aa43-e4791626d2f1.png)

**Closing thoughts**
Could it be that the developers of the tool intentionally disabled UIAutomation so that it wouldnt work with their application? is that even possible?

Thank you in advance and have a nice day.

**Comment by Huntk23** [comment]
I just started using this library to automate a lot of crazy dynamic WinForm applications and have run into a ton of weird scenarios. One comes to mind here, have you attempted something like:

```cs
var descendants = bomgarAuthenticationDialog.FindAllDescendants();

foreach (var descendant in descendants)
{
    descendant.DrawHighlight();
}
```
Seeing if the text/edit boxes actually get highlighted? There is a chance you are not getting the text box you are after with `d.ByControlType(ControlType.Edit)` because it may be a child descendant of a pane or other control. Meaning you may need to get the descendants of a descendant.

For example:
```cs
var controlPane = bomgarAuthenticationDialog.FindFirstDescendant(d => d.ByControlType(ControlType.Pane));
var textBox = controlPane.FindFirstDescendant(d => d.ControlType(ControlType.Edit)).AsTextBox();

textBox.Enter("Test");
```

Basically, try and just look at the tree of controls that is found with `bomgarAuthenticationDialog.FindAllDescendants();`
